packages/realm-react/CHANGELOG.md

```diff
@@ -4,13 +4,29 @@
 * None
 
 ### Enhancements
-* None
+* Added the ability to get `progress` information in `fallback` component of `RealmProvider` when opening a synced Realm. ([#6785](https://github.com/realm/realm-js/issues/6785))
+```tsx
+import { RealmProvider, RealmProviderFallback } from "@realm/react";
+
+const Fallback: RealmProviderFallback = ({ progress }) => {
+	return <Text>Loading:{(100 * progress).toFixed()}%</Text>;
+}
+
+const MyApp() = () => {
+  return (
+    <RealmProvider sync={...} fallback={Fallback}> 
+      ...
+    </RealmProvider>
+  );
+}
+```
 
 ### Fixed
 * <How to hit and notice issue? what was the impact?> ([#????](https://github.com/realm/realm-js/issues/????), since v?.?.?)
 * None
 
 ### Compatibility
+* Realm JavaScript >= v12.12.0
```

Comment:
Should we add a note at the top (under `## vNext (TBD)`) about the bump of the Realm JS version?

Reply:
Will merge this and change changelog in the useProgress hook PR